src/wallet/rpcdump.cpp

```diff
@@ -86,7 +86,8 @@ UniValue importprivkey(const JSONRPCRequest& request)
             "1. \"privkey\"          (string, required) The private key (see dumpprivkey)\n"
             "2. \"label\"            (string, optional, default=\"\") An optional label\n"
             "3. rescan               (boolean, optional, default=true) Rescan the wallet for transactions\n"
-            "\nNote: This call can take minutes to complete if rescan is true.\n"
+            "\nNote: This call can take minutes to complete if rescan is true, during that time, other rpc calls\n"
+            "may report that the imported key exists but related transactions are still missing.\n"
```

Comment:
You should probably add a ", leading to temporarily incorrect/bogus balances and unspent outputs until rescan completes" to the warning message.

Reply:
Fixed by adding @TheBlueMatt's proposed sentence-part.